manifests/overlays/prod/configs/argo_cm/resource.inclusions

```diff
@@ -6,6 +6,18 @@
   - https://api.thoth01.lab.upshift.rdu2.redhat.com:6443
   - https://api.thoth12.lab.upshift.rdu2.redhat.com:6443
   - https://api.ocp4.idh.dev.datahub.redhat.com:6443
+- apiGroups:
+  - '*'
```

Comment:
the wildcard here might cause issues, can you specify the apigroups individually?

Reply:
good point, updated the api groups.